pyproject.toml

```diff
@@ -95,3 +81,45 @@ torchseg = ["py.typed"]
 
 [tool.setuptools.packages.find]
 include = ["torchseg*"]
+
+[tool.ruff]
+exclude = [
+    # Extra files
+    "data/",
+    "images/",
+    "logo/",
+    "logs/",
+    "output/",
+
+    # Docs
+    "docs/src/",
+
+    # Spack
+    ".spack-env/",
+
+    # Python
+    "build/",
+    "dist/",
+    ".cache/",
+    ".mypy_cache/",
+    ".pytest_cache/",
+    "__pycache__/",
+    "*.egg-info/",
+
+    # Git
+    ".git/",
+    ".github/",
```

Comment:
Are you sure ruff doesn't ignore most of these by default?

Reply:
It actually does because it checks the gitignore. I can remove these